scripts/manifest/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+    "abseil": {
+        "name": "abseil-cpp",
+        "sha": "273292d1cfc0a94a65082ee350509af1d113344d",
+        "tag": "20220623.0",
+        "url": "https://github.com/abseil/abseil-cpp.git"
+    },
+    "cares": {
+        "name": "c-ares",
+        "sha": "6360e96b5cf8e5980c887ce58ef727e53d77243a",
+        "tag": "v1.19.1",
+        "url": "https://github.com/c-ares/c-ares.git"
+    },
+    "cctz": {
+        "label": "master~25",
+        "name": "cctz",
+        "sha": "02918d62329ef440935862719829d061a5f4beba",
+        "url": "https://github.com/google/cctz.git"
+    },
+    "gflags": {
+        "label": "master~5",
+        "name": "gflags",
+        "sha": "827c769e5fc98e0f2a34c47cef953cc6328abced",
+        "url": "https://github.com/gflags/gflags.git"
+    },
+    "glog": {
+        "name": "glog",
+        "sha": "a8e0007e96ff96145022c488e367da10f835c75d",
+        "tag": "v0.6.0-rc1",
+        "url": "https://github.com/google/glog.git"
+    },
+    "grpc": {
```

Comment:
Defaults have been updated with the merge of #41 Please upate version, SHA and tag